<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QWidget" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonThumb1">
           <property name="text">
            <string>Thumbnails</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonEdit">
           <property name="text">
            <string>Edit</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonEditSave">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonEditCancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSliderZoom">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>504</width>
            <height>642</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageThumb">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButtonThumb2">
           <property name="text">
            <string>Thumbnails</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSlider" name="horizontalSliderZoom_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QScrollArea" name="scrollAreaThumb">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContentsThumb">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>504</width>
            <height>642</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonThumb1</sender>
   <signal>clicked(bool)</signal>
   <receiver>Viewer</receiver>
   <slot>thumbnails(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonEditCancel</sender>
   <signal>clicked()</signal>
   <receiver>Viewer</receiver>
   <slot>cancelEditForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>496</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonEdit</sender>
   <signal>clicked()</signal>
   <receiver>Viewer</receiver>
   <slot>editForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderZoom</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Viewer</receiver>
   <slot>setZoom(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonThumb2</sender>
   <signal>toggled(bool)</signal>
   <receiver>Viewer</receiver>
   <slot>thumbnails(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>448</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonEditSave</sender>
   <signal>clicked()</signal>
   <receiver>Viewer</receiver>
   <slot>saveEditForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setZoom(int)</slot>
  <slot>butt()</slot>
  <slot>zoomPressed()</slot>
  <slot>setPage(int)</slot>
  <slot>cancelEditForm()</slot>
  <slot>thumbnails(bool)</slot>
  <slot>editForm()</slot>
  <slot>saveEditForm()</slot>
 </slots>
</ui>
